packages/dev/docs/pages/architecture.mdx

```diff
@@ -0,0 +1,262 @@
+tables, usually have very similar behavior and logic. There are even 
+specifications that describe how many of the most common components should
+behave in terms of [accessibility semantics](https://www.w3.org/TR/wai-aria-1.2/)
+and [keyboard interactions](https://www.w3.org/TR/wai-aria-practices-1.2/).
+The main difference between design systems is the styling.
+
+This opens up to the possibility of sharing much of the behavior and 
+component logic between design systems and across platforms. For example,
+user interactions, accessibility, internationalization, and behavior
+can be reused, while allowing custom styling and rendering to live
+within individual design systems. This has the potential to improve
+the overall quality of applications, while saving companies money
+and time, and reducing duplicated effort across the industry.
+
+## Architecture
+
+<figure className={style.float}>
+  <Flex flexDirection="column" gap="size-125">
+    <View backgroundColor="orange-400" height="size-900">
+      <Flex alignItems="center" justifyContent="center" height="100%">
+        <span style={{color: 'white', fontSize: 18}}>Component</span>
+      </Flex>
+    </View>
+    <View backgroundColor="chartreuse-400" height="size-900">
+      <Flex alignItems="center" justifyContent="center" height="100%">
+        <span style={{color: 'white', fontSize: 18}}>Behavior Hook</span>
+      </Flex>
+    </View>
+    <View backgroundColor="blue-400" height="size-900">
+      <Flex alignItems="center" justifyContent="center" height="100%">
+        <span style={{color: 'white', fontSize: 18}}>State Hook</span>
+      </Flex>
+    </View>
+  </Flex>
+</figure>
+
+In order to allow reusing component behavior between design systems,
+React Spectrum splits each component into three parts: state, behavior,
+and the rendered component.
+
+Some components don't have all of these pieces. For example, some simple 
+components do not require any state, and others are only compositions of 
+other components, so each component has the pieces of this architecture 
+that make sense for their purpose.
+
+This architecture is made possible by 
+[React Hooks](https://reactjs.org/docs/hooks-intro.html), which enable
+the ability to reuse behavior between multiple components. Hooks allow
+accessing React features like state and effects from functions which
+can be called from any component. If you're unfamiliar with hooks,
+we recommend reading the documentation linked above first.
+
+### State hook
+
+At the bottom is the state hook. This hook is shared across platforms —
+it could work on the web, react-native, or any other platform, and makes
+not assumption about the view system it is running on. It also has no 
+theme or design system specific logic.
+
+The state hook accepts common props from the component and provides 
+state management. It returns an interface for reading and updating
+component state, with methods that implement much of the core logic
```

Comment:
i think this comma is extraneous? the sentence feels awkward though, I can't quite tell if you're saying with as in, in addition to, or if you're saying the updating of the component state is done with...
Maybe, "It implements the core logic for the component and returns an interface for reading and updating the component state." ?